src/test/scala/org/ergoplatform/local/MempoolAuditorSpec.scala

```diff
@@ -80,7 +80,7 @@ class MempoolAuditorSpec extends AnyFlatSpec with NodeViewTestOps with ErgoTestH
 
     getPoolSize shouldBe 1 // first tx removed from pool during node view update
 
-    scorex.core.utils.untilTimeout(cleanupDuration * 3, 100.millis) {
+    scorex.core.utils.untilTimeout(cleanupDuration * 4, 100.millis) {
```

Comment:
@kushti You probably know but this test often fails due to busy Travis. It did not help though